examples/micro-ros_publisher/transport.cpp

```diff
@@ -0,0 +1,50 @@
+#include <Arduino.h>
```

Comment:
I would love to be able to change the transport in the project instead of the library. MCUs usually have multiple UARTs that users may switch to, or to other protocols, SPI, I2C, and CAN. Also, we can benefit from Arduino's libraries to interact with those protocols. Defining transport in the project would be super handy.

However, in that case, we need to include the same transport file in all examples which is not the best solution. Let me know if you have any suggestions.